src/plugins/visualizations/public/plugin.ts

```diff
@@ -138,11 +138,11 @@ export class VisualizationsPlugin
   private readonly types: TypesService = new TypesService();
   private getStartServicesOrDie?: StartServicesGetter<VisualizationsStartDeps, VisualizationsStart>;
 
-  constructor(initializerContext: PluginInitializerContext) {}
+  constructor(_initializerContext: PluginInitializerContext) {}
```

Comment:
Why these changes here?
Seems a bit out of scope, did I miss anything?

Reply:
Yes I agree on that, why have we changed that?

Reply:
You're right that this isn't related to the functionality being introduced in this PR. I was looking at this file and my editor flagged this parameter as well as an unused local variable below.

Typescript [uses the underscore in front](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals) to denote a parameter that isn't intended to actually be used. Since we don't run our builds with the strict `noUnusedParameters` rule, this case doesn't break the build. But, adding the underscore fixes my typescript intellisense.

I have no strong feelings about this change, so let me know if you'd like it reverted.

Reply:
If you can revert it that would be better, just to make this PR cleaner.
We can do a follow up PR with all these cleaning tasks later.

Reply:
Done.